bootstrap/peer.go

```diff
@@ -194,8 +184,11 @@ func (peer *Peer) Run(ctx context.Context) error {
 	})
 	group.Go(func() error {
 		// TODO: move the message into Server instead
-		peer.Log.Sugar().Infof("Node %s started on %s", peer.Identity.ID, peer.Public.Server.Addr().String())
-		return ignoreCancel(peer.Public.Server.Run(ctx))
+		// Don't change the format of this comment, it is used to figure out the node id.
+		peer.Log.Sugar().Infof("Node %s", peer.Identity.ID)
```

Comment:
Does storj-sim still show node id-s in log? I suspect this might have broken due to the format change.

Reply:
sorry, I tried to see how storj-sim was parsing and thought this syntax was fine.  Let me check again and I can revert the comment change.

Reply:
storj-sim uses a really stupid approach. it chceks whether log line starts with "Node " and searches until " started". So if you add ` started` after the `%s` it should work.

Reply:
ah ok, I will make that change.